extensions/adobe/experience/adcloud-experienceevent.example.2.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
There is no measure in this example, we need to have one measure. I know that the example.1 also lacks this and we may want to fix that here as well.

Maybe this
  "xdm:advertising": {
    "xdm:clicks": {
      "xdm:value": 1.0
    }
  }

Reply:
My commit resolved this